oat-sa/tao-item-runner-qti-fe
# Replace input to div

Related to: https://oat-sa.atlassian.net/browse/MS-579

**Changes**

In text entry inline interactions `input` element was changed to `div` for `reviewRenderer` to support highlight plugin.

**How to test?**

- Create a test with text entry inline interaction
- Take the test on tao
- Login as a scorer on MS to review the responses
- All the text entry inline interactions should be displayed as expected and should have been rendered in a `div` instead of `input`
